src/dmd/root/ctfloat.d

```diff
@@ -41,6 +41,8 @@ extern (C++) struct CTFloat
   @nogc:
   @safe:
 
+    alias isNaN = .isNaN;
```

Comment:
why?

Reply:
Backwards compatibility. You don't like large diffs. By moving the function out of the struct it's possible to use the UFCS syntax to call the function.